src/components/organisms/FileTreePane/TreeItem.tsx

```diff
@@ -74,9 +74,8 @@ const TreeItem: React.FC<TreeItemProps> = props => {
 
   const isRoot = treeKey === ROOT_FILE_ENTRY;
   const root = fileMap[ROOT_FILE_ENTRY];
-  const node = fileMap[treeKey];
```

Comment:
For future reference, this bug occurred because `fileMap` does not include excluded files, yet we have an option to Show/Hide excluded files. `node` will be `undefined` in this case and it will crash.

Reply:
This assumption is incorrect and a new fix has been pushed that fixes the actual issue - an outdated `tree`.